front/pages/api/v1/w/[wId]/vaults/[vId]/apps/[aId]/runs/[runId]/index.ts

```diff
@@ -0,0 +1,122 @@
+/* eslint jsdoc/no-missing-syntax: 0 */ //
+// Disabling jsdoc rule, as we're not yet documentating dust apps endpoints under vaults.
+// We still document the legacy endpoint, which does the same thing.
+// Note: for now, an API key only has access to the global vault.
+import type { RunType, WithAPIErrorResponse } from "@dust-tt/types";
```

Comment:
`nit`: IIRC we usually do this:

https://github.com/dust-tt/dust/blob/edab8258b0c22fa484f1b56c36937639196bd81b/front/pages/api/v1/w/%5BwId%5D/feature_flags.ts#L15-L18

Reply:
TIL thx